App/Config/ReduxPersist.js

```diff
@@ -66,14 +66,14 @@ const REDUX_PERSIST = {
   reducerVersion: '1.0',
   storeConfig: {
     key: 'primary',
-    version: 4,
+    version: 5,
```

Comment:
I don't think you need to increment this if you're not providing any migration.

Reply:
undone